src/auth/authMessages.js

```diff
@@ -47,14 +55,15 @@ type AuthMetadata = {
  * Use this to pass options that aren't part of the Blockstack auth spec, but might be supported
  * by special authenticators.
  * @return {String} the authentication request
+ * @private
  */
-export function makeAuthRequest(transitPrivateKey: string = generateAndStoreTransitKey(),
```

Comment:
This is a breaking change - some apps use `makeAuthRequest`, and it's in the public API. For example, we use this function when telling people how to use the custom Gaia onboarding flow. I'd suggest keeping the method name the same.

Reply:
This is addressed in https://github.com/blockstack/blockstack.js/pull/542/commits/db940b247d0ef7bef24c618bb75ad6c6f04726b5